jibri/rootfs/defaults/autoscaler-sidecar.config

```diff
@@ -0,0 +1,14 @@
+export PORT={{ .Env.AUTOSCALER_SIDECAR_PORT }}
```

Comment:
This port is not present in the compose ports / expose section. Does it need to? Also no default?

Reply:
It's only used internally by the jibri running in the same container.  It's default is set in the shell script